cmd/launcher/interactive.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/kolide/kit/env"
+	"github.com/kolide/kit/fs"
+	"github.com/kolide/launcher/pkg/osquery/table"
+	osquery "github.com/osquery/osquery-go"
+	"github.com/pkg/errors"
+)
+
+func interactive(args []string) error {
```

Comment:
```suggestion
func runInteractive(args []string) error {
```

slight preference for naming